client/multi-currency-setup/tasks/add-currencies-task/test/utils.test.js

```diff
@@ -0,0 +1,172 @@
+/**
+ * Internal dependencies
+ */
+import {
+	ConcatenateCurrencyStrings,
+	StringRepresentationOfCurrency,
+} from '../utils';
+
+const availableCurrencies = {
+	USD: {
+		code: 'USD',
+		rate: 1,
+		name: 'United States (US) dollar',
+		id: 'usd',
+		is_default: true,
+		flag: '🇺🇸',
+		symbol: '$',
+	},
+	CAD: {
+		code: 'CAD',
+		rate: '1.206823',
+		name: 'Canadian dollar',
+		id: 'cad',
+		is_default: false,
+		flag: '🇨🇦',
+		symbol: '$',
+	},
+	GBP: {
+		code: 'GBP',
+		rate: '0.708099',
+		name: 'Pound sterling',
+		id: 'gbp',
+		is_default: false,
+		flag: '🇬🇧',
+		symbol: '£',
+	},
+	EUR: {
+		code: 'EUR',
+		rate: '0.826381',
+		name: 'Euro',
+		id: 'eur',
+		is_default: false,
+		flag: '🇪🇺',
+		symbol: '€',
+	},
+	AED: {
+		code: 'AED',
+		rate: '3.6732',
+		name: 'United Arab Emirates dirham',
+		id: 'aed',
+		is_default: false,
+		flag: '🇦🇪',
+		symbol: 'د.إ',
+	},
+	CDF: {
+		code: 'CDF',
+		rate: '2000',
+		name: 'Congolese franc',
+		id: 'cdf',
+		is_default: false,
+		flag: '🇨🇩',
+		symbol: 'Fr',
+	},
+	AUD: {
+		code: 'AUD',
+		rate: 1.79,
+		name: 'Australian dollar',
+		id: 'aud',
+		is_default: false,
+		flag: '🇦🇺',
+		symbol: '$',
+	},
+	JPY: {
+		code: 'JPY',
+		rate: 1,
+		name: 'Japanese yen',
+		id: 'jpy',
+		is_default: false,
+		flag: '🇯🇵',
+		symbol: '¥',
+	},
+	INR: {
+		code: 'INR',
+		rate: 1,
+		name: 'Indian rupee',
+		id: 'inr',
+		is_default: false,
+		flag: '🇮🇳',
+		symbol: '₹',
+		is_zero_decimal: false,
+		last_updated: 1630070442,
+	},
+	DKK: {
+		code: 'DKK',
+		rate: '6.144615',
+		name: 'Danish krone',
+		id: 'dkk',
+		is_default: false,
+		flag: '🇩🇰',
+		symbol: 'DKK',
+	},
+	BIF: {
+		code: 'BIF',
+		rate: '1974',
+		name: 'Burundian franc',
+		id: 'bif',
+		is_default: false,
+		flag: '🇧🇮',
+		symbol: 'Fr',
+	},
+	CLP: {
+		code: 'CLP',
+		rate: '706.8',
+		name: 'Chilean peso',
+		id: 'clp',
+		is_default: false,
+		flag: '🇨🇱',
+		symbol: '$',
+	},
```

Comment:
Don't want to be too picky 😅 But we can get rid of the unused currencies properties in the test:
```suggestion
	USD: {
		code: 'USD',
		name: 'United States (US) dollar',
		symbol: '$',
	},
	CAD: {
		code: 'CAD',
		name: 'Canadian dollar',
		symbol: '$',
	},
	GBP: {
		code: 'GBP',
		name: 'Pound sterling',
		symbol: '£',
	},
	EUR: {
		code: 'EUR',
		name: 'Euro',
		symbol: '€',
	},
	AED: {
		code: 'AED',
		name: 'United Arab Emirates dirham',
		symbol: 'د.إ',
	},
	CDF: {
		code: 'CDF',
		name: 'Congolese franc',
		symbol: 'Fr',
	},
	AUD: {
		code: 'AUD',
		name: 'Australian dollar',
		symbol: '$',
	},
	JPY: {
		code: 'JPY',
		name: 'Japanese yen',
		symbol: '¥',
	},
	INR: {
		code: 'INR',
		name: 'Indian rupee',
		symbol: '₹',
	},
	DKK: {
		code: 'DKK',
		name: 'Danish krone',
		symbol: 'DKK',
	},
	BIF: {
		code: 'BIF',
		name: 'Burundian franc',
		symbol: 'Fr',
	},
	CLP: {
		code: 'CLP',
		name: 'Chilean peso',
		symbol: '$',
	},
```

Reply:
Sorry, I merged it before seeing this comment 😮 I guess they'll stay 😬